spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+- {layer: 'pois', kind: 'lifeguard_tower'}
+- {layer: 'pois', kind: 'mast'}
+- {layer: 'pois', kind: 'maze'}
+- {layer: 'pois', kind: 'memorial'}
+- {layer: 'pois', kind: 'mooring'}
+# reserve for kind_details
+- _reserved: {from: 4211, to: 4231}
+- {layer: 'pois', kind: 'motorcycle_parking'}
+- {layer: 'pois', kind: 'petroleum_well'}
+- {layer: 'pois', kind: 'platform'}
+- {layer: 'pois', kind: 'pylon'}
+- {layer: 'pois', kind: 'rock'}
+- {layer: 'earth', kind: 'earth'}
+- {layer: 'landuse', kind: 'apron'}
+- {layer: 'landuse', kind: 'substation'}
+- {layer: 'landuse', kind: 'summer_toboggan'}
+- {layer: 'landuse', kind: 'plant'}
+- {layer: 'landuse', kind: 'pitch'}
+- {layer: 'landuse', kind: 'animal'}
+- {layer: 'landuse', kind: 'artwork'}
+- {layer: 'landuse', kind: 'carousel'}
+- {layer: 'landuse', kind: 'water_slide'}
+- {layer: 'landuse', kind: 'parking'}
+- {layer: 'water', kind: 'fountain'}
+- {layer: 'water', kind: 'ditch'}
+- {layer: 'water', kind: 'drain'}
+- {layer: 'water', kind: 'swimming_pool'}
+- {layer: 'landuse', kind: 'breakwater'}
+- {layer: 'landuse', kind: 'cutline'}
+- {layer: 'landuse', kind: 'hedge'}
+- {layer: 'landuse', kind: 'tree_row'}
+- {layer: 'landuse', kind: 'retaining_wall'}
+- {layer: 'landuse', kind: 'snow_fence'}
+- {layer: 'landuse', kind: 'fence'}
+# reserve for kind_details
+- _reserved: {from: 4259, to: 4309}
+- {layer: 'landuse', kind: 'wall'}
+# reserve for kind_details
+- _reserved: {from: 4311, to: 4331}
+- {layer: 'landuse', kind: 'power_minor_line'}
+- {layer: 'landuse', kind: 'power_line'}
+- {layer: 'transit', kind: 'bus_stop'}
+- {layer: 'buildings', kind: 'building_part'}
+# reserve for kind_details
+- _reserved: {from: 4336, to: 4386}
+- {layer: 'pois', kind: 'atm'}
+- {layer: 'pois', kind: 'bus_stop'}
+- {layer: 'pois', kind: 'firepit'}
+- {layer: 'pois', kind: 'life_ring'}
+- {layer: 'pois', kind: 'picnic_table'}
+- {layer: 'pois', kind: 'shower'}
+- {layer: 'pois', kind: 'taxi'}
+- {layer: 'pois', kind: 'telephone'}
+- {layer: 'pois', kind: 'toilets'}
+# reserve for kind_details
+- _reserved: {from: 4396, to: 4406}
+- {layer: 'pois', kind: 'waste_disposal'}
+- {layer: 'pois', kind: 'bbq'}
+- {layer: 'pois', kind: 'bicycle_repair_station'}
+- {layer: 'pois', kind: 'cycle_barrier'}
+- {layer: 'pois', kind: 'drinking_water'}
+- {layer: 'pois', kind: 'phone'}
+- {layer: 'buildings', kind: 'address'}
+- {layer: 'pois', kind: 'bench'}
+- {layer: 'buildings', kind: 'entrance'}
+# reserve for kind_details
+- _reserved: {from: 4416, to: 4436}
+- {layer: 'buildings', kind: 'exit'}
+# reserve for kind_details
+- _reserved: {from: 4438, to: 4448}
+- {layer: 'pois', kind: 'fire_hydrant'}
+- {layer: 'pois', kind: 'gas_canister'}
+- {layer: 'pois', kind: 'level_crossing'}
+- {layer: 'pois', kind: 'love_hotel'}
+- {layer: 'pois', kind: 'post_box'}
+- {layer: 'pois', kind: 'power_pole'}
+- {layer: 'pois', kind: 'street_lamp'}
+- {layer: 'pois', kind: 'traffic_signals'}
```

Comment:
I got `traffic_signals` one wrong... let's move it just above the `atm` row, please.

Reply:
Moved in ba8e791.